src/rnp/rnp.c

```diff
@@ -315,10 +347,12 @@ rnp_cmd(rnp_cfg_t *cfg, rnp_t *rnp, int cmd, char *f)
         ctx.sigcreate = get_birthtime(rnp_cfg_get(cfg, CFG_BIRTHTIME));
         ctx.sigexpire = get_duration(rnp_cfg_get(cfg, CFG_DURATION));
 
+        clearsign = (cmd == CMD_CLEARSIGN) ? true : false;
```

Comment:
@flowher I believe I changed it to bool (among with other things in rnp.c/rnpcfg.c) in some other branch which did not become pull request yet :) Sorry, was a bit out of rnp this week, will get back tomorrow.
@ronaldtse everything should be fixed here, just was waiting for approvals.